### PR TITLE
Add Gitlab integration

### PR DESCRIPTION
**Pull request by olhado**
This allows the user to define a PagerDuty schedule that deputize will take the users from and assign to a Gitlab group.

This can be useful for ensuring there is someone in the oncall rotation that can approve merge requests for changes, without pulling in other people.

### CODE REVIEW

File: oncall/oncall.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // deputize - Update an LDAP group with info from the PagerDuty API
 // oncall.go: On Call Updater
 //
-// Copyright 2017 Threat Stack, Inc. All rights reserved.
+// Copyright 2017-2020 Threat Stack, Inc. All rights reserved.
 // Author: Patrick T. Cable II <pat.cable@threatstack.com>
```
Comment (patcable) [inline oncall/oncall.go:5]: Add yourself in here @olhado! :)

File: oncall/oncall.go [hunk 2]
```diff
@@ -168,6 +174,96 @@ func UpdateOnCallRotation(conf config.DeputizeConfig) error {
...
+      log.Printf("No new Approvers, not updating Gitlab group: %s", conf.GitlabGroup)
+    } else {
+      // Add OnCall approvers to approver group
+      if !conf.Quiet {
+        log.Printf("Updating Gitlab group: %s", conf.GitlabGroup)
+      }
+
+      // Remove existing members of the group, if they exist
+      if !conf.Quiet {
+        log.Printf("Removing old approvers %s from Gitlab group: %s", conf.GitlabGroup)
```
Comment (patcable) [inline oncall/oncall.go]: missing two vars here

Comment (olhado) [reply oncall/oncall.go]: Actually that was only supposed to have 1. Thanks for the catch!

File: oncall/oncall.go (hunk 1 above)
Comment (olhado) [reply oncall/oncall.go:5]: Done.

**Pull request merged**